docOps/docusaurus.config.js

```diff
@@ -0,0 +1,111 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'iota.rs',
+  tagline: 'Official IOTA iota.rs Software',
```

Comment:
This sounds a little bit weird with IOTA iota, how about `Official IOTA Client Software`?